src/BloomExe/HtmlThumbNailer.cs

```diff
@@ -265,7 +265,10 @@ private bool OpenTempFileInBrowser(GeckoWebBrowser browser, string filePath)
 			{
 				_isolator.Navigate(browser, filePath); // main thread, not async, we really need it now.
 				while (!order.Done)
+				{
 					Application.DoEvents();
+					Application.RaiseIdle(new EventArgs());
```

Comment:
Let's add comment that it is needed for Linux, else in the future it could be removed by someone noticing that it is "not needed".